app/models/candidate.rb

```diff
@@ -48,7 +48,11 @@ def current_application
   end
 
   def current_application_choice
```

Comment:
Can this method be dropped now, if it’s no longer referenced in `OfferValidations`? I can't think of anywhere where picking the last choice by date created would be needed – although we should check?

Reply:
Good shout — it is kind of a weird method anyway!?